src/views/Predictions/components/RoundCard/SetPositionCard.tsx

```diff
@@ -169,7 +155,7 @@ const SetPositionCard: React.FC<SetPositionCardProps> = ({ position, togglePosit
   }, [value, maxBalance, minBetAmount, setErrorMessage, t])
 
   return (
-    <Card onMouseOver={handleMouseOver} onMouseOut={handleMouseOut}>
```

Comment:
When back button is tapped onMouseOver triggered on mobile which causes to disable and detach events, swiper-no-swiping class is a better approach to disable swiper events where we don't want.